docs/QUICK-START.md

```diff
@@ -0,0 +1,23 @@
+# Quick Start Network Service Mesh
+
+This document assumes you already have a kubernetes cluster, and all packages such as golang, docker and kubernetes. If not,
```

Comment:
This approach should not require golang to install. The binaries and images are pre-built and pulled from the docker hub.

Reply:
I'll merge it in and add a new PR changing this.